met-web/src/components/layout/SideNav/UserGuideNav.tsx

```diff
@@ -63,9 +66,46 @@ const UserGuideNav = () => {
     };
 
     return (
-        <ListItem key="user-guide">
-            <ListItemButton onClick={openHelpPage}>
-                <MetHeader4 color="white">User Guide</MetHeader4>
+        <ListItem
+            key="user-guide"
+            sx={{
+                backgroundColor: Palette.background.default,
+                '&:hover, &:focus': {
+                    filter: 'brightness(96%)',
+                },
+                '&:active': {
+                    filter: 'brightness(92%)',
+                },
+            }}
+        >
+            <ListItemButton
+                onClick={openHelpPage}
+                disableRipple
+                sx={{
+                    '&:hover, &:active, &:focus': {
+                        backgroundColor: 'transparent',
+                    },
+                }}
+            >
+                <FontAwesomeIcon
+                    icon={faBookOpen}
+                    style={{
+                        fontSize: '1.1rem',
+                        color: Palette.text.primary,
+                        paddingRight: '0.75rem',
+                        width: '1.1rem',
+                    }}
+                />
+                <MetHeader4
```

Comment:
Folks using assistive technologies will expect headers to denote a section to follow. If you just used an <h4> to make the text large and prominent, then you should use a link tag and style it to be large and prominent instead.

Also I'd strongly recommend we update the `openHelpPage` function to open the link in the same window. Using new windows/tabs can confuse some users and are discouraged unless preserving the current context of the page is absolutely necessary: https://www.w3.org/WAI/WCAG21/Techniques/general/G200